packages/react/src/useFormSubmission.js

```diff
@@ -172,7 +174,9 @@ const useFormSubmission = (
 
   return {
     formChangeState,
+    getAllFieldValues,
     getErrorMessages,
+    getErrorMessagesByField,
```

Comment:
I'd like to also see `getRegisteredFieldValues` and the deprecation of `getErrorMessages` since `getErrorMessagesByField` is the correct implementation I meant to do in the first place.

I'd also like to see `getErrorMessagesByField` changed to `getErrors` and converted to an array of objects with an `errorMessages` and `fieldName` prop. Makes it easier for consumers to loop and group it. That's going to be the most-likely use case.

Reply:
Done, though doing an `Object.entries(...).map(...)` on an object is easier than reconstituting the object from the array :D

Reply:
How would you do the deprecation? Right now, I've deprecated it in Typescript (but that's nothing more than a warning in the editor while trying to use it), but couldn't find how you deprecate things. I usually go for `console.warn`, but you might think that excessive.

Reply:
I have a deprecation utility in one of my micro-service libraries, but it's Node.js-only.

I don't mind the TypeScript deprecation. I can also deprecate or remove it in the docs.

Since this is a brand new feature, I honestly don't think anyone was using it; although, who knows! 🤷‍♂️. Just trying to be safe as a library author.